packages/core/src/styles/components/_toolbar.scss

```diff
@@ -0,0 +1,80 @@
+$css--reset: false;
+@import "./../vendor/carbon-components/src/components/overflow-menu/overflow-menu";
+
+g.#{$prefix}--#{$charts-prefix}--toolbar {
+	@include carbon--motion(standard, expressive);
+
+	.toolbar-container {
+		.toolbar-button {
+			fill: $ui-background;
+			cursor: pointer;
+			&:hover {
```

Comment:
```suggestion

			&:hover {
```

Reply:
Updated in https://github.com/carbon-design-system/carbon-charts/pull/791/commits/56b3c20d8ac2990a06938ac3cd80d0d54c8bde20.